source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -781,24 +812,9 @@ socket:
   be enough when image contains lots of bitmaps.  with defaults, each
   bitmap uses 2M per TB (engine, vdsm).
 
-- Are bitmaps copied when copying images with qemu-img convert? if not
-  next backup after LSM or move disk must be full (qemu-img).
-
-- Are bitmaps copied in block copy during LSM for the active layer?
-  If not the next backup must be full (libvirt/qemu).
-
-- How to handle hot plug disk? can we continue to track changes on a
-  detached disk, or we should require a full backup in this case (libvirt)?
-
 - Attaching disk with bitmaps to older qemu version will invalidate
   bitmaps, creating corrupted backups. how can we prevent this? (qemu).
 
-- Are bitmaps are copied down from the active layer during live merge
-  (libvirt/qemu) and cold merge (qemu-img)?
-
-- Need libvirt API to list all bitmaps in an image in a running VM
-  (libvirt).
-
```

Comment:
Same, don't mix unrelated changes with the API change.

Reply:
restored all the unrelated notes, I will remove them in a separate patch